saleor/urls.py

```diff
@@ -17,6 +18,11 @@
         digital_product,
         name="digital-product",
     ),
+    url(
```

Comment:
why not path() or re_path here?

Reply:
to keep consistency with the above patterns ☝️ . I know that `url()` will be deprecated in the future but I think that migrating to `path` or `re_path` should be done in one and separate PR